Amazon.Ion.ObjectMapper.Test/IonSerializerTest.cs

```diff
@@ -35,5 +36,16 @@ public void SerializesAndDeserializesLists()
         {
             Check(new int[] { 1, 1, 2, 3, 5, 8, 11 });
         }
+
+        [TestMethod]
+        public void AnnotatedIonSerializer()
+        {
+            var annotatedIonSerializer = new Dictionary<string, dynamic>();
+            annotatedIonSerializer.Add("Supra", new SupraSerializer());
+            var serializer = new IonSerializer(new IonSerializationOptions { AnnotatedIonSerializers = annotatedIonSerializer });
+            var stream = serializer.Serialize(TestObjects.a90);
+            var output = serializer.Deserialize<string>(stream);
+            Assert.AreEqual(TestObjects.a90.ToString(), output);
+        }

```

Comment:
Add more tests to cover multiple serializers and edge cases.

Reply:
Done.